apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -778,6 +779,30 @@ import { FavoriteQuery } from '@/common/appdb/models/favorite_query'
       },
       fakeRemoteChange() {
         this.query.text = "select * from foo"
+      },
+      getColumnsForAutocomplete() {
+        const cm = this.editor.getValue() 
+        if (cm.toLowerCase().search(/[from(\s)?|join(\s)?]/g) === -1) return
```

Comment:
Is this how sqlhint identifies tables? Can we steal that logic instead?

This won't work with aliases, updates, deletes, etc.